ambry-api/src/main/java/com/github/ambry/config/FrontendConfig.java

```diff
@@ -50,6 +52,13 @@
   private static final String DEFAULT_ACCOUNT_STATS_STORE_FACTORY =
       "com.github.ambry.accountstats.InmemoryAccountStatsStoreFactory";
 
+  private static final String DEFAULT_CONTAINER_METRICS_ENABLED_REQUEST_TYPES =
+      "{\"DeleteBlob\", \"GetBlob\", \"GetBlobInfo\", \"GetSignedUrl\", \"PostBlob\", \"UpdateBlobTtl\", \"UndeleteBlob\", \"PutBlob\"}";
```

Comment:
I think this default value needs to just be a comma separated list for the string.split code in RestRequestMetricsGroup to work.

Reply:
Yeah, I've updated the code.